manifests/03-rbac-role-ns-openshift-console-user-settings.yaml

```diff
@@ -0,0 +1,45 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-user-settings-role-operator
```

Comment:
Lets drop the `-operator` suffix. It's a bit confusing since there is actually no operator that would be handling the `openshift-console-user-settings` namespace.
Also most of the time we are not adding the `-role` suffix as well since the resource is `Role` itself.
In case we want a name that would reflect the role, I would rather go with `console-user-settings-admin`